### PR TITLE
Custom headers

### PR DESCRIPTION
**Pull request by weinong**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### CODE REVIEW

File: src/aks-preview/azext_aks_preview/_help.py [hunk 1]
```diff
@@ -181,6 +181,9 @@
         - name: --enable-managed-identity
           type: bool
           short-summary: (PREVIEW) Using a system assigned managed identity to manage cluster resource group.
+        - name: --aks-custom-headers
```
Comment (yangl900) [inline src/aks-preview/azext_aks_preview/_help.py]: "--custom-headers"?

Comment (weinong) [reply src/aks-preview/azext_aks_preview/_help.py]: unfortunately you cant. `--custom-headers` seems to be a reserved keyword used by other thing. when i used that long time ago, i couldn't get the argument to pass in

**Comment by fengzhou-msft** [comment]
please take a look at the travis log and fix the issue.

**Comment by weinong** [comment]
@fengzhou-msft fixed. thanks!

File: src/aks-preview/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,3 @@
+FROM microsoft/azure-cli:latest
```
Comment (fengzhou-msft) [inline src/aks-preview/Dockerfile]: azure-cli docker is migrated to [MCR](https://hub.docker.com/_/microsoft-azure-cli), please use `mcr.microsoft.com/azure-cli:latest`

Comment (weinong) [reply src/aks-preview/Dockerfile]: good catch. I actually dont need that file. :)

File: src/aks-preview/create-5.sh [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+export SUBSCRIPTION_ID=c1089427-83d3-4286-9f35-5af546a6eb67
+export LOCATION=eastus2euap
+export RESOURCE_GROUP_NAME=weinongw-${LOCATION}
+export AKS_NAME=weinong-p-
+export VNET_NAME=weinongw-vnet
+export AKS_SUBNET_NAME=aks
+export AKS_VERSION=1.14.6
+export AKS_SPN_CLIENT_ID=13bec9da-7208-4aa0-8fc7-47b25e26ff5d
+export AKS_SPN_SECRET=2aQUC72zjh_C8+C.]kSdfKGjrH52DZ[Z
```
Comment (haroldrandom) [inline src/aks-preview/create-5.sh]: hard code secret can not get CI job CredScan pass.

Please don't hard code any secret or password.

Comment (weinong) [reply src/aks-preview/create-5.sh]: sorry about that. i removed it and reset the password. :(

**Comment by weinong** [comment]
@haroldrandom @fengzhou-msft can we merge this soon?

**Pull request merged**